docs/release-workflow.md

```diff
@@ -0,0 +1,15 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Release workflow
+
+This document explains the steps and timings of the release process in `tezos-packaging`.
+
+Releasing process is almost fully automated, the only steps that require manual
+interaction are the review of two PRs created by the CI.
```

Comment:
I don't think it necessarily makes sense to fully transcribe what the diagram says, but I'd like at least some more info to be in text, e.g. for people using screenreaders (I'm not sure if the .svg text is supported there), or just for information at a glance. Let's maybe add just another line here about how long it usually takes, e.g. 

```suggestion
Releasing process is almost fully automated, with the initial release PR being created
at most 4 hours after a new Octez release. The only steps that require manual intervention
are reviews of this initial PR and of the PR that updates macOS brew formulae.
This allows `tezos-packaging` to closely follow Octez releases without large waiting times.
```

What do you think? 🤔

Reply:
That makes sense to mention in the doc as well